package/debian/DEBIAN/control

```diff
@@ -1,5 +1,5 @@
 Package: easee
-Version: 0.0.9
+Version: 2.0.5
```

Comment:
Just a comment, it should be tagged as 2.1.0 as SIgnalR and other are new features.